submissions/tests/test_serializers.py

```diff
@@ -1,32 +1,59 @@
 """
 Tests for submissions serializers.
 """
+import ddt
 from django.test import TestCase
-from submissions.models import Score, StudentItem
+from submissions.models import Score, ScoreAnnotation, StudentItem, Submission
 from submissions.serializers import ScoreSerializer
 
 
+@ddt.ddt
 class ScoreSerializerTest(TestCase):
     """
     Tests for the score serializer.
     """
 
-    def test_score_with_null_submission(self):
-        item = StudentItem.objects.create(
+    def setUp(self):
+        super(ScoreSerializerTest, self).setUp()
+        self.item = StudentItem.objects.create(
             student_id="score_test_student",
             course_id="score_test_course",
             item_id="i4x://mycourse/special_presentation"
         )
+        self.submission = Submission.objects.create(student_item=self.item, attempt_number=1)
+
+        self.score = Score.objects.create(
+            student_item=self.item,
+            submission=self.submission,
+            points_earned=2,
+            points_possible=6,
+        )
 
+
+    def test_score_with_null_submission(self):
         # Create a score with a null submission
-        score = Score.objects.create(
-            student_item=item,
+        null_sub_score = Score.objects.create(
+            student_item=self.item,
             submission=None,
-            points_earned=2,
-            points_possible=6
+            points_earned=3,
+            points_possible=8,
         )
-        score_dict = ScoreSerializer(score).data
+        null_sub_score_dict = ScoreSerializer(null_sub_score).data
+        self.assertIs(null_sub_score_dict['submission_uuid'], None)
+        self.assertEqual(null_sub_score_dict['points_earned'], 3)
+        self.assertEqual(null_sub_score_dict['points_possible'], 8)
```

Comment:
Because there was only 1 test method in this file before, I took the liberty of modifying it to use a common `setUp` method. Then I wanted the specialized score being tested here to be defined differently than the generalized `self.score`, which is why it's 3/8 instead of 2/6.